core/src/main/java/io/questdb/griffin/FunctionParser.java

```diff
@@ -276,6 +283,9 @@ public void visit(ExpressionNode node) throws SqlException {
                 case ExpressionNode.QUERY:
                     functionStack.push(createCursorFunction(node));
                     break;
+                case ExpressionNode.GEOHASH_TYPE_SIZE:
```

Comment:
whoa, this is intense. ExpressionNode cannot be a type, we must keep this constant. Lets regroup tomorrow.